CHANGELOG.md

```diff
@@ -3,4 +3,5 @@
 
 BUG FIXES:
 
-* Fix token renewals when TokenPolicies have been configured [[GH-82](https://github.com/hashicorp/vault-plugin-auth-gcp/pull/82)]
\ No newline at end of file
+* Fix token renewals when TokenPolicies have been configured [[GH-82](https://github.com/hashicorp/vault-plugin-auth-gcp/pull/82)]
+* Add BATs acceptance tests for Vault GCP Auth [GH-122](https://github.com/hashicorp/vault-plugin-auth-gcp/pull/122)
```

Comment:
I feel like this doesn't need a changelog entry since it is just adds acceptance tests that are transparent to the user. But if it does need a changelog entry, I don't think it should go under "bug fixes".

Reply:
Removed in `c89a8e9`